bokeh/events.py

```diff
@@ -146,43 +206,103 @@ def __init__(self, model, sx=None,sy=None, x=None, y=None):
+
     '''
     event_name = 'doubletap'
 
 class Press(PointEvent):
     ''' Announce a press event on a Bokeh plot.
 
+    Attributes:
+        sx (float) : x-coordinate of the event in *screen* space
+        sx (float) : y-coordinate of the event in *screen* space
+        x (float) : x-coordinate of the event in *data* space
+        y (float) : y-coordinate of the event in *data* space
+
     '''
     event_name = 'press'
 
-
 class MouseEnter(PointEvent):
     ''' Announce a mouse enter event onto a Bokeh plot.
 
+    Attributes:
+        sx (float) : x-coordinate of the event in *screen* space
+        sx (float) : y-coordinate of the event in *screen* space
+        x (float) : x-coordinate of the event in *data* space
+        y (float) : y-coordinate of the event in *data* space
+
+    .. note::
+        The enter event is generated when the mouse leaves the entire Plot
+        canvas, including any border padding and space for axes or legends.
+
     '''
     event_name = 'mouseenter'
 
 class MouseLeave(PointEvent):
     ''' Announce a mouse leave event from a Bokeh plot.
 
+    Attributes:
+        sx (float) : x-coordinate of the event in *screen* space
+        sx (float) : y-coordinate of the event in *screen* space
+        x (float) : x-coordinate of the event in *data* space
+        y (float) : y-coordinate of the event in *data* space
+
+    .. note::
+        The leave event is generated when the mouse leaves the entire Plot
+        canvas, including any border padding and space for axes or legends.
+
     '''
     event_name = 'mouseleave'
 
 class MouseMove(PointEvent):
     ''' Announce a mouse movement event over a Bokeh plot.
 
+    Attributes:
+        sx (float) : x-coordinate of the event in *screen* space
+        sx (float) : y-coordinate of the event in *screen* space
+        x (float) : x-coordinate of the event in *data* space
+        y (float) : y-coordinate of the event in *data* space
+
+    .. note::
+        This event can fire at a very high rate, potentially increasing network
+        traffic or CPU load.
+
     '''
     event_name = 'mousemove'
 
 class MouseWheel(PointEvent):
-    ''' Announce a mouse whee event on a Bokeh plot.
+    ''' Announce a mouse wheel event on a Bokeh plot.
+
+    Attributes:
+        delta (float) : the (signed) scroll speed
+        sx (float) : x-coordinate of the event in *screen* space
+        sx (float) : y-coordinate of the event in *screen* space
+        x (float) : x-coordinate of the event in *data* space
+        y (float) : y-coordinate of the event in *data* space
+
+
+    By default, Bokeh plots do not capture scroll events, because prevents
+    the page from scrolling which can be annoying unless it was explicitly
+    requested. In order for ``MouseWheel`` events to fire, either the
+    ``WheelZoomTool`` must be active, or some custom extension must have
+    requested wheel UI events to propagate.
```

Comment:
replaced with:

By default, Bokeh plots do not prevent default scroll events unless a WheelZoomTool or WheelPanTool is active. This may change in future releases.